File: atividades_verde/C/parOuImpar.c
#include <stdio.h>

void parOuImpar(int x) {
    if(x % 2 == 0) {
        printf("P\n");
    }
    else {
        printf("I\n");
    }
}

int main() {
    int num;
    do
    {   
        scanf("%d", &num);
        if(num != 0) {
            parOuImpar(num);
        }
    } while (num != 0);
    
    return 0;
}

File: workspace/atividades_verde/JAVA/ParOuImpar.java
public class ParOuImpar {

    private static void parOuImpar(int x) {
        if(x % 2 == 0) {
            System.out.println("P");
        }
        else {
            System.out.println("I");
        }
    }
    public static void main(String[] args) {
        int num;
        do {
            num = MyIO.readInt();
            if(num != 0) {
                parOuImpar(num);
            }
        } while(num != 0);
    }
}